maths/Kadane's_algorithm.py

```diff
@@ -0,0 +1,69 @@
+"""
+    Kadane's algorithm to get maximum subarray sum
+    Please enter the decimal values only and use spaces as seperator

```

Comment:
Please add URL to a website that explains the algorithm and why it would be useful.

Reply:
Thanks for the suggestions
As I am new to open source contribution this help is meant a lot to me.